test/powershell/engine/Help/assets/HelpURI/V2Cmdlets.csv

```diff
@@ -151,7 +151,6 @@ Out-Null,https://go.microsoft.com/fwlink/?linkid=2096792
 Out-Printer,https://go.microsoft.com/fwlink/?LinkID=2109553
 Out-String,https://go.microsoft.com/fwlink/?linkid=2097024
 Pop-Location,https://go.microsoft.com/fwlink/?linkid=2096907
```

Comment:
```suggestion
Pop-Location,https://go.microsoft.com/fwlink/?linkid=2096907
Prompt,https://go.microsoft.com/fwlink/?LinkId=2136843
```

Reply:
Added new link

Reply:
@yecril71pl We discussed this internally. The decision was made to point to the about topic.

Reply:
Please enlighten me regarding the prevailing argument.

Reply:
1. There is a function named `prompt`
2. It makes sense for the help information for said function to point to information about prompts.

Is it so unclear?

Reply:
It is unclear is why it should happen in the online version only.